examples/plugin/settings.gradle

```diff
@@ -0,0 +1,8 @@
+rootProject.name = 'plugin-example'
+
+// There are no published ORT binaries that this example could depend on, so use a Gradle composite build.
+includeBuild('../..') {
```

Comment:
I'm wondering whether we really need to use a composite build here. Couldn't we make `plugin` are regular sub-project of the root project, and move dependency substitution to the plugin project's `build.gradle`, like in https://docs.gradle.org/current/userguide/customizing_dependency_resolution_behavior.html#sec:project_to_module_substitution.